src/main.ts

```diff
@@ -2,7 +2,9 @@ import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
 import { Amplify } from 'aws-amplify';
 
-require('dotenv').config;
+require('dotenv').config({
+  path: '../.env',

```

Comment:
duda existencial, por qué no usan el import al estilo es6? 
```suggestion
import dotenv from "dotenv"
dotenv.config({})

```

Reply:
buen punto ahora lo pongo 👍🏼